src/gui/vector/qgsattributesformproperties.cpp

```diff
@@ -678,6 +700,11 @@ void QgsAttributesFormProperties::mEditorLayoutComboBox_currentIndexChanged( int
       mAddContainerButton->setVisible( false );
       mRemoveLayoutItemButton->setVisible( false );
       mInvertSelectionButton->setVisible( false );
+
+      // Workaround to match the allignment of the
+      // Available Widgets panel with the horizontal row
+      // holding the search box (when Form Layout disappears)
+      searchBarLayout->setContentsMargins( 0, 0, 6, 0 );
```

Comment:
An image explains it better. This is what we avoid with this workaround:

![Screenshot from 2025-04-24 10-16-38](https://github.com/user-attachments/assets/2684964c-9add-4a70-ae2e-b5bfcbe9bc72)